Proposal to WG LC comments

haomianzheng opened the pull request:
This update captures the corresponding changes according to the LC comments of ietf-layer0-types. 
This update also captures the comments on the list: https://mailarchive.ietf.org/arch/msg/ccamp/dURl9mDv5ObaH2v7eo5lckjG1tw/
The remained point 3 in the list above is not solved yet, and tracked as: 
https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-wson-yang/issues/2
This update also solved the open issue: https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-wson-yang/issues/1

haomianzheng committed:
Add files via upload

This update captures the corresponding changes according to the LC comments of ietf-layer0-types. 
This update also captures the comments on the list: https://mailarchive.ietf.org/arch/msg/ccamp/dURl9mDv5ObaH2v7eo5lckjG1tw/
The remained point 3 in the list above is not solved yet, and tracked as: 
https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-wson-yang/issues/2

italobusi commented on ietf-wson-topology.yang:
```diff
@@ -84,21 +83,16 @@ module ietf-wson-topology {
-          base layer0-types:layer0-node-type;
-        }
-        description "WSON node type";
+      description "Candidate type: ROADM or FOADM";
+      leaf is-reconfigurable-node {
+        type boolean;
+        description 
+		  "true - ROADM, 
+		   false - FOADM. ";
```
See my comment: https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-wson-yang/issues/1#issuecomment-613953152

```suggestion
      presence "The TE node is a WSON node.";
      description "WSON node attributes";
      leaf is-reconfigurable-node {
        type boolean;
        default true;
        description
          "Indicates whether the WSON node is reconfigurable:
           - true: the node is reconfigurable
             (e.g., it is representing a ROADM node);
           - false: the node is not reconfigurable
             (e.g., it is representing a FOADM node).";
```

italobusi commented on ietf-wson-topology.yang:
```diff
@@ -233,318 +156,11 @@ module ietf-wson-topology {
-  }
-
   /*
-   * Augment TE bandwidth
+   * Augment TE bandwidth is removed to be solved in next version. 
    */
```
I am not sure whether we need or not to add these augmentations in the next version. Better to remove this comment.

italobusi reviewed (changes requested):
See my comments below

Please also note that the YANG model and tree are using Windows EOL. In order to facilitate change reviews, it is proposed to change the EOL conventions to UNIX only after this PR has been merged.

haomianzheng committed:
Update ietf-wson-topology.yang

This commit addresses the following issue: 
https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-wson-yang/issues/2

haomianzheng committed:
Update ietf-wson-topology.yang

Co-Authored-By: italobusi <italo.busi@huawei.com>

haomianzheng committed:
Update ietf-wson-topology.yang

italobusi commented on ietf-wson-topology.yang:
```diff
@@ -69,7 +68,7 @@ module ietf-wson-topology {
      (http://trustee.ietf.org/license-info).";
 
-  revision 2019-02-27 {
+  revision 2020-04-15 {
```
```suggestion
  revision 2020-04-20 {
```

italobusi commented on ietf-wson-topology.yang:
```diff
@@ -84,108 +83,18 @@ module ietf-wson-topology {
   grouping wson-node-attributes {
```
```suggestion

  grouping wson-node-attributes {
```

italobusi reviewed (approved):
Just two minor editorial suggestions

haomianzheng committed:
Update ietf-wson-topology.yang

Co-Authored-By: italobusi <italo.busi@huawei.com>

haomianzheng committed:
Update ietf-wson-topology.yang

Co-Authored-By: italobusi <italo.busi@huawei.com>

haomianzheng committed:
Add files via upload

Update the date and some indentation.
